docs/api/exceptions.rst

```diff
@@ -0,0 +1,39 @@
+
+.. xdata:: datatable.exceptions
```

Comment:
`datatable.exceptions` is a module, not a variable. Similar to `datatable.math` or `datatable.models`.

Reply:
Yeah, typo. So how do we define modules now? `.. py:module:: datatable.exceptions` or `.. xpy:module:: datatable.exceptions`. For some reason we have both now for math and models.